Public/Src/Cache/ContentStore/Library/Service/Grpc/GrpcCopyClientCache.cs

```diff
@@ -0,0 +1,146 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using BuildXL.Cache.ContentStore.Exceptions;
+using BuildXL.Cache.ContentStore.Interfaces.Results;
+using BuildXL.Cache.ContentStore.Interfaces.Tracing;
+
+namespace BuildXL.Cache.ContentStore.Service.Grpc
+{
+    /// <summary>
+    /// Cache for <see cref="GrpcCopyClient"/>.
+    /// </summary>
+    public sealed class GrpcCopyClientCache
+    {
+        // Link to DistributedContentStoreSettings.MaxConcurrentCopyOperations
+        internal readonly ConcurrentDictionary<GrpcCopyClientKey, GrpcCopyClient> _clientDict;

```

Comment:
Minor, but I would prefer to keep it internal add an internal method that will do what you want this field for.